server.go

```diff
@@ -45,16 +45,16 @@ type Quote struct {
 	CryptoKey   string
 }
 
-func getQuote(commandString string, UserId string) (string, error) {
-	//	Check if the user exists, and if they have the necessary balance to request a quote
-	stockSymbol := strings.Split(commandString, ",")[1]
-
+func getQuote(stockSymbol string, userId string, transactionNum int) (Quote, error) {
 	quoteTime := int64(time.Nanosecond) * int64(time.Now().UnixNano()) / int64(time.Millisecond)
 
+	commandString := stockSymbol + "," + userId
+
 	if cachedQuote, exists := quoteMap[stockSymbol]; exists {
 		if cachedQuote.Timestamp+60000 > quoteTime {
-			quoteString := cachedQuote.Price + "," + cachedQuote.StockSymbol + "," + cachedQuote.UserId + "," + strconv.FormatInt(cachedQuote.Timestamp, 10) + "," + cachedQuote.CryptoKey
-			return quoteString, nil
+			auditCacheEvent := SystemEvent{Server: SERVER, Command: "QUOTE", StockSymbol: stockSymbol, Username: userId, Filename: FILENAME, Funds: floatStringToCents(cachedQuote.Price), TransactionNum: transactionNum}
```

Comment:
Not sure if Command should be QUOTE, or if we should add another parameter to getQuote and then make it the command that asked for the quote

may or may not resolve #18